agent/wptdriver/web_driver.cc

```diff
@@ -73,6 +73,9 @@ bool WebDriver::RunAndWait() {
     active_event = CreateMutex(&null_dacl, TRUE, GLOBAL_TESTING_MUTEX);
   }
 
+  // Delete all scripts from the scripts directory, just in case there are left-overs

```

Comment:
Are we deleting the script elsewhere? If not, we should not talk about left-overs.